x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_config_panel.tsx

```diff
@@ -161,6 +162,37 @@ export function XYConfigPanel(props: VisualizationProps<State>) {
         </EuiFormRow>
       )}
 
+      <EuiFormRow
+        label={i18n.translate('xpack.lens.xyChart.splitSeries', {
+          defaultMessage: 'Split series',
+        })}
+      >
+        <>
+          {state.splitSeriesAccessors.map(columnId => (
```

Comment:
There should be a way to remove existing split column like for y columns.

Reply:
You can:

![image](https://user-images.githubusercontent.com/833377/60359648-9f237500-99a7-11e9-81c9-f73872d0ead0.png)

That's the split series list.

Reply:
@chrisdavies That just deletes the column in the data source, not the accessor in the visualization. If you press the plus button ten times, you have ten columns "to configure" and now way to remove those placeholders.
<img width="274" alt="Screenshot 2019-06-28 at 19 27 44" src="https://user-images.githubusercontent.com/1508364/60359968-d2332c00-99da-11e9-9a4d-1d8f0d0b0203.png">

y axis has separate remove buttons but they are not styled nicely
<img width="271" alt="Screenshot 2019-06-28 at 19 28 06" src="https://user-images.githubusercontent.com/1508364/60359999-e70fbf80-99da-11e9-83f8-dc58e47aa352.png">